recruiter/models.py

```diff
@@ -10,6 +10,19 @@ class Company(models.Model):
     def __str__(self):
         return self.name
 
+    def get_all_recruiters(self):
```

Comment:
Why do we need this function? Which role does it fulfill within the application's business logic? the function name and documentation should indicate that.

By the time you are done writing your business logic I should be able to get a full picture of what your application will be like on a logical level - which input data it takes, what outputs it generates and which processes is it doing.

Reply:
We want this function In case a student wants to get a list of all recruiters of a specified company
I will change the name to make the function more clear